library/src/lib/calendar/calendar2/calendar2-views/calendar2-month-view/calendar2-month-view.component.ts

```diff
@@ -1,17 +1,104 @@
-import { Component, OnInit, ViewEncapsulation } from '@angular/core';
+import { Component, Input, Output, ViewEncapsulation, EventEmitter, ElementRef } from '@angular/core';
 
 @Component({
     selector: 'fd-calendar2-month-view',
     templateUrl: './calendar2-month-view.component.html',
     styleUrls: ['./calendar2-month-view.component.scss'],
     encapsulation: ViewEncapsulation.None
 })
-export class Calendar2MonthViewComponent implements OnInit {
+export class Calendar2MonthViewComponent {
+    /** @hidden The id of the calendar passed from the parent component. */
```

Comment:
you can remove all the @hidden tags since we won't be displaying this component page to the users at all